containers/jetty/src/main/java/com/yahoo/athenz/container/AthenzJettyContainer.java

```diff
@@ -351,8 +327,8 @@ void loadServicePrivateKey() {
         PrivateKeyStoreFactory pkeyFactory;
         try {
             pkeyFactory = (PrivateKeyStoreFactory) Class.forName(pkeyFactoryClass).newInstance();
-        } catch (InstantiationException | IllegalAccessException | ClassNotFoundException e) {
-            LOG.error("Invalid PrivateKeyStoreFactory class: {} error: {}", pkeyFactoryClass, e.getMessage());
+        } catch (InstantiationException | IllegalAccessException | ClassNotFoundException ex) {
+            LOG.error("Invalid PrivateKeyStoreFactory class: {}", pkeyFactoryClass, ex);
```

Comment:
Log only has one {} (the ex will be ignored)

Reply:
it won't be. if the last argument is an exception, then it includes the full backtrace automatically. I replaced the message with the full trace since sometimes we just get a null message and would have preferred to see the full backtrace in the log to see what was the failure.